RoslynPluginGenerator/AnalyzerPluginGenerator.cs

```diff
@@ -94,15 +91,22 @@ public bool Generate(NuGetReference analyzeRef, string language, string sqaleFil
                 this.logger.LogDebug(UIResources.APG_CreatedTempWorkingDir, baseDirectory);
 
                 // Collect the remaining data required to build the plugin
+                RoslynPluginDefinition definition = new RoslynPluginDefinition();
+                definition.Language = language;
+                definition.SqaleFilePath = sqaleFilePath;
                 definition.PackageId = package.Id;
                 definition.PackageVersion = package.Version.ToString();
                 definition.Manifest = CreatePluginDefinition(package);
                 definition.RulesFilePath = Path.Combine(baseDirectory, "rules.xml");
-                definition.EmbeddedZipFileName = package.Id + ".zip"; // TODO - generate the zip file
```

Comment:
The zip file is being created as follows: the local NuGet repository has unzipped the files to a single directory. I've changed the NuGet handler interface to return this directory, and we just zip it again.
However, the zip contains the original .nupkg file, which is in itself a zip file. I'll change this is a later commit.